org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
are you sure? gtk2 does not support wayland.

Reply:
Wouldn't that get routed through Xwayland internally anyway?

Reply:
no. 

Did you even try it?